### ISSUE TITLE
loadFromJSON - canvas image 16000px width @ 600 DPI renders shorter then is was

### ISSUE THREAD

**Issue by edmaybert** [fabricjs/fabric.js#7310]
I have a Fabric canvas onto which I place a PNG with the following resolution:
16001px X 2676px @ 600dpi

I scale the image to fit the width of the canvas. ( 8.5 X 11 inches ) and save.

When I restore the canvas via loadFromJSON, the image is no longer the width of the canvas. It's shorter ( width-wise ).

I'm trying to figure out why this is happening.

Any thought as to why would be greatly appreciated.
~ed

**Comment by asturur** [comment]
hi @edmaybert when the image has a size in pixels DPI are not that much relevant.
Can you share any code snippet, or can you reproduce it outside your application in a js fiddle?

**Comment by edmaybert** [comment]
Yeah. I noticed adjusting the DPI had no real effect. But reducing the image width resolved it. 
I will trey to post a fiddler and see if I can reproduce the error.
